src/Tribe/Customizer/Text.php

```diff
@@ -0,0 +1,272 @@
+<?php
+/**
+ * The Events Calendar Customizer Section: Text.
+ *
+ * @package The Events Calendar
+ * @since   TBD
+ */
+
+// Don't load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+final class Tribe__Events__Customizer__Text extends Tribe__Customizer__Section {
+
```

Comment:
* :no_entry_sign: Missing doc comment for class Tribe__Events__Customizer__Text
* :no_entry_sign: Class name is not valid; consider Tribe_Events_Customizer_Text instead

<sub>hash: b7db5bddac4b5a087c8f676ed1fdb5c5</sub>